obspy/core/util/base.py

```diff
@@ -573,7 +573,10 @@ def download_to_file(url, filename_or_buffer, chunk_size=1024):
     except TypeError:
         r = requests.get(url)
 
-    r.raise_for_status()
+    # Raise anything except for 200
+    if r.status_code != 200:
+        raise requests.HTTPError(u'%s HTTP Error: %s for url: %s'
```

Comment:
It the `u` prefix necessary here?

Reply:
Nah probably not. I copied it in from `requests.raise_for_status()`.